### ISSUE TITLE
The virtual.db and transport.db files are created empty

### ISSUE THREAD

**Issue by gothicx** [camptocamp/puppet-postfix#130]
RHEL 7.2:

```
Notice: /Stage[main]/Postfix::Mta/Postfix::Hash[/etc/postfix/virtual]/File[/etc/postfix/virtual.db]/ensure: created
Notice: /Stage[main]/Postfix::Mta/Postfix::Hash[/etc/postfix/transport]/File[/etc/postfix/transport.db]/ensure: created

-rw-r--r--. 1 root root  13K Jan 26  2014 transport
-rw-------. 1 root root    0 Apr 28 14:25 transport.db
-rw-r--r--. 1 root root  13K Jan 26  2014 virtual
-rw-------. 1 root root    0 Apr 28 14:25 virtual.db

```

**Comment by shollingsworth** [comment]
Having the same problem...

**Comment by alexjfisher** [comment]
This is a duplicate of https://github.com/camptocamp/puppet-postfix/issues/101 ?

**Comment by gothicx** [comment]
It looks like..

**Comment by mimmus** [comment]
Same problem on RH 7.2

**Comment by raphink** [comment]
#135 was merged. Does it fix this?

**Comment by mimmus** [comment]
Does #135  was merged in 1.3.1?

**Comment by raphink** [comment]
@mimmus no, it's not released yet.
